.ci/build-n-run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Change the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to `#!/usr/bin/env bash`, which allows the overriding.